hello-world-mtls/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "build": "tsc --build",
+    "build.watch": "tsc --build --watch",
+    "start": "ts-node src/worker.ts",
+    "start.watch": "nodemon src/worker.ts",
+    "workflow": "ts-node src/exec-workflow.ts"
+  },
+  "devDependencies": {
+    "@tsconfig/node16": "^1.0.0",
```

Comment:
Other examples use node 14. Do you think we should upgrade to node 16 across the board? Node 16 isn't quite the active LTS yet, but it will be as of October 26: https://nodejs.org/en/about/releases/

Reply:
Yeah, I think we're recommending 16 in general—Roey changed setup docs and nvmrc's to be 16.

Reply:
Let's switch to 16, it's close enough to LTS.

Reply:
Thinking about it some more, it'd be nice to detect the user's node version when the script runs and use the correct tsconfig.

Reply:
Okay, so we can leave them as node16 here in case people manually clone, and have `@temporalio/create` replace it before `npm install`ing if user's node version is different.

Reply:
Implemented in https://github.com/lorensr/sdk-node/commit/3740f7728005d9f0cd6c18293788581b23944e59